app/balancer/management/commands/discord_bot.py

```diff
@@ -131,7 +131,7 @@ async def on_button_click(interaction: discord.Interaction, button):
             player = Command.get_player_by_name(interaction.user.name)
 
             if not player and type != 'register_form':
-                await interaction.channel.send(f'`{interaction.user.name}`: I don\'t know him')
+                await interaction.channel.send(f'`{interaction.user.name}`: Nieznany, niezarejestrowany gracz')
```

Comment:
tłumaczenia hard-coded na polski odpadają zupełnie. Preferowana forma to: import jsona do arrayki, a w kodzie odwoływanie się do kluczów array'ki. W ten sposób jednym JSONem zewnętrznym umożliwimy tłumaczenie nie tylko na PL, ale na wszystkie języki, byleby JSON się zgadzał. Snippet do obsługi tego w podstawowej formie wysłałem na DC w priv